dojo/components/views.py

```diff
@@ -0,0 +1,22 @@
+from django.shortcuts import render
+from dojo.models import Finding
+from django.db.models import Count, Q
+from dojo.utils import add_breadcrumb, get_page_items
+from dojo.filters import ComponentFilter
+
+
+def components(request):
+    add_breadcrumb(title='Components', top_level=True, request=request)
+    component_query = Finding.objects.filter().values("component_name", "component_version")
+    component_query = component_query.order_by('component_name', 'component_version').distinct()
```

Comment:
is the distinct() really needed?

Reply:
Tested and it doesn't make difference. Removed in last commit.